pom.xml

```diff
@@ -1140,12 +1175,11 @@
                 <jdk>[9,)</jdk>
             </activation>
             <properties>
-                <maven-surefire-plugin.argLine.add-modules>--add-modules=${maven-surefire-plugin.jigsaw.modules}</maven-surefire-plugin.argLine.add-modules>
```

Comment:
Is the add-modules option not necessary anymore?

Reply:
Yes, we don't need it anymore as we removed our JAXB dependency: as it's totally removed from JDK 11, we needed to move to Stax (or add a specific dependency but that's not what we decided).